backend/danswer/connectors/confluence/utils.py

```diff
@@ -18,7 +18,25 @@
 
 logger = setup_logger()
 
-_USER_NOT_FOUND = "Unknown User"
+
+_USER_EMAIL_CACHE: dict[str, str | None] = {}
+
+
```

Comment:
Might consider a cache decorator instead

Reply:
unfortunately there are issues with caching complicated objects like confluence_connector. We believe this was causing the memory leak